packages/docsearch-react/src/DocSearchModal.tsx

```diff
@@ -139,6 +139,7 @@ export function DocSearchModal({
 
             return [
               {
+                sourceId: 'DocSearchRecentSearches',
```

Comment:
From algolia/autocomplete#490 we'll recommend naming those as `recentSearches` to pick the `sourceId`s easily with destructuring. We can start naming those following this convention already.

Reply:
Make sense!

Reply:
Fixed in https://github.com/algolia/docsearch/pull/999/commits/fc965daeb4877a3febd619012dff39febc7f5de7